### PR TITLE
Upgrade to the latest Bref beta version

### PR DESCRIPTION
**Pull request by mnapoli**
Watch out this involves large changes 😄 I open this to open the discussion about moving to typed events.

I realized that by using the now native `SqsHandler` and `SnsHandler` abstract classes, we can get rid of the `Consumer` interface as well as type resolvers (the type detection is now done by bref).

All of this applies the constraint of having 1 event per lambda (can't have a lambda listen to SQS and SNS), but it seems to me like a good thing: those things scale very differently, should have different error behaviors, etc.

If we go this way, there is no need for the BrefWorker class as well: each handler can use directly the appropriate consumer class.

WDYT?

### CODE REVIEW

File: bin/consumer.php [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-lambda(function ($event) {
-    $kernel = new \App\Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
-    $kernel->boot();
+$kernel = new \App\Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
+$kernel->boot();
 
-    $worker = $kernel->getContainer()->get(BrefWorker::class);
-    $worker->consumeLambdaEvent($event);
-});
+return $kernel->getContainer()->get(SqsConsumer::class);
```
Comment (mnapoli) [inline bin/consumer.php]: I realize now that this file can be removed right?

File: composer.json [hunk 2]
```diff
@@ -22,7 +22,8 @@
         "symfony/config": "^4.3 || ^5.0",
         "symfony/dependency-injection": "^4.3 || ^5.0",
         "symfony/http-kernel": "^4.3 || ^5.0",
-        "symfony/yaml": "^4.3 || ^5.0"
+        "symfony/yaml": "^4.3 || ^5.0",
+        "bref/bref": "^0.5.13@beta"
```
Comment (mnapoli) [inline composer.json]: TODO: wait for stable release.

**Comment by Nyholm** [comment]
Thank you for this PR. I'll have to try it out all look at the result locally. 

(It feels great to remove code =))

**Comment by Nyholm** [comment]
I really like removing a lot of code =)

I think this makes sense. I've been using this package more and more now. We are not done, but this is a good step. I'll be happy to merge after a rebase and remove of `@beta`.

**Comment by mnapoli** [comment]
👍  rebased and uses now a stable Bref release.

**Comment by Nyholm** [comment]
Thank yo

**Pull request merged**